charts/dex/templates/ingress.yaml

```diff
@@ -1,9 +1,6 @@
 {{- if .Values.ingress.enabled -}}
 {{- $fullName := include "dex.fullname" . -}}
 {{- $svcPort := .Values.service.ports.http.port -}}
-{{- if .Values.https.enabled -}}
-  {{- $svcPort = .Values.service.ports.https.port -}}
-{{- end }}
```

Comment:
It's sad but since this ticket is not done upstream https://github.com/dexidp/helm-charts/issues/15
We need to restore this "manual patch"

Reply:
Good catch! Btw, we should likely consider a more predictable way for applying these patches, maybe something similar to what they do in https://github.com/scality/Zenko